### PR TITLE
Node length

### PR DESCRIPTION
**Pull request by JaspervanTilburg**

### CODE REVIEW

File: src/main/java/gui/GraphDrawer.java [hunk 1]
```diff
@@ -90,39 +95,57 @@ public void moveShapes(double xDifference) {
         drawEdges();
     }
 
+    public void initializeColumnWidths() {
```
Comment (Jipje) [inline src/main/java/gui/GraphDrawer.java]: Looks good!

**Comment by Jipje** [comment]
The go to a node function doesn't work with the new radius definition.

**Comment by JaspervanTilburg** [comment]
Go to node function works again

**Comment by Jipje** [comment]
Okay looks good, lets solve those conflicts and we're ready to go.

**Pull request merged**